communication/mavlink_stream.c

```diff
@@ -52,21 +52,42 @@
 // PUBLIC FUNCTIONS IMPLEMENTATION
```

Comment:
I comment here, because it concerns lines that were not changed. 

I believe we can remove the includes to onboard_parameters.h and mavlink_message_handler.h, at least when I remove them, it still compiles and I don't see the functions that needs these include.